### PR TITLE
Render member's biography and photo in the member display page

### PR DESCRIPTION
**Pull request by scudmissile**
This is to work in conjunction with the API changes https://github.com/DSASanFrancisco/membership_api/pull/6

### CODE REVIEW

**Comment by scudmissile** [comment]
@jeffmay I've rebased this branch onto master and updated the package-lock.json file.

File: src/js/components/membership/Member.jsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { Link } from 'react-router'
 import AddRole from '../admin/AddMeeting'
 import AddMeeting from '../admin/AddRole'
 import AddEligibleVoter from '../admin/AddEligibleVoter'
+var ReactMarkdown = require('react-markdown');
```
Comment (jeffmay) [inline src/js/components/membership/Member.jsx]: Why not use `import` here?

File: package-lock.json [hunk 2]
```diff
@@ -4,4161 +4,4824 @@
   "lockfileVersion": 1,
   "dependencies": {
     "abbrev": {
-      "version": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.0.tgz",
+      "version": "1.1.0",
+      "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.0.tgz",
```
Comment (jeffmay) [inline package-lock.json]: I keep seeing this switch back and forth. I think we need to standardize on our version of npm to fix this.

File: src/index.html [hunk 3]
```diff
@@ -3,6 +3,7 @@
 <html>
 <head>
   <meta http-equiv="content-type" content="text/html; charset=UTF8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (jeffmay) [inline src/index.html]: @stevengoldberg didn't you already add this? https://github.com/DSASanFrancisco/membership_ui/pull/8/files#diff-e249faefed5757034596c5096d33dab6R6

Comment (stevengoldberg) [inline src/index.html]: Yes, looks like `master` needs to be merged into this branch.

File: package-lock.json (hunk 2 above)
Comment (stevengoldberg) [inline package-lock.json]: I think this may be the same issue with working from a stale version of `master`.

**Comment by jeffmay** [comment]
I vote we see if https://github.com/DSASanFrancisco/membership_ui/pull/15 has what we need from this PR, and if so, just close this one.

**Comment by scudmissile** [comment]
@jeffmay Thanks! https://github.com/DSASanFrancisco/membership_ui/pull/15 has all my changes but a cleaner history.

File: src/js/components/membership/Member.jsx (hunk 1 above)
Comment (scudmissile) [inline src/js/components/membership/Member.jsx]: I tried using `import` at first, but it wasn't working for me. I'm not sure if I was using an incorrect `import` statement or if react-markdown isn't exporting a module appropriately for `import`, but `require` worked so I went with that.

**Pull request closed without merging**